test/python3/libdnf5/rpm/test_package_query.py

```diff
@@ -124,7 +124,8 @@ def test_resolve_pkg_spec(self):
         # Test passing an explicit list of forms
         query = libdnf5.rpm.PackageQuery(self.base)
         settings = libdnf5.base.ResolveSpecSettings()
-        settings.nevra_forms.append(libdnf5.rpm.Nevra.Form_NA)
+        settings.set_nevra_forms(
+            libdnf5.rpm.VectorNevraForm(1, libdnf5.rpm.Nevra.Form_NA))
```

Comment:
This just've stumped me for a moment... Is this a standard way we are creating vector instances in Python bindings?

Reply:
If I understand it correctly it could be made to work with just a Python list but we would need a Swig `typemap` for the nevra forms. Otherwise I think this is the only way.

Reply:
OK I see, we could eventually improve it later.